### PR TITLE
Clear support list of kiwix-serve supported browsers

### PR DESCRIPTION
**Pull request by kelson42**
Fixes https://github.com/kiwix/libkiwix/issues/1029

### CODE REVIEW

**Comment by kelson42** [comment]
@veloman-yunkan Any thoughts on this PR?

File: docs/kiwix-serve.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ file).
 Clients can also remotely search inside those ZIM files that contain a full-text
 search database.
 
+``kiwix-serve`` supports Web browsers `Firefox >= 70, Chrome >= 80, Edge >= 80, ChromeAndroid >= 80, Safari >= 14, iOS >= 14 <https://browsersl.ist/#q=Firefox+%3E%3D+70%2C+Chrome+%3E%3D+80%2C+Edge+%3E%3D+100%2C+ChromeAndroid+%3E%3D+80%2C+Safari+%3E%3D+14%2C+iOS+%3E%3D+14>`_.
```
Comment (veloman-yunkan) [inline docs/kiwix-serve.rst:20]: Inconsistency for the Edge browser - documentation states `Edge >= 80`, the link shows `Edge >= 100`.

Comment (kelson42) [reply docs/kiwix-serve.rst]: Fixed!

**Review by veloman-yunkan — APPROVED** [review]
Please squash the commits before merging

**Pull request merged**